CONTRIBUTING.md

```diff
@@ -344,14 +344,15 @@ description: {
 
 }
 ```
+(Note: if you choose to use `changelog-d`'s Markdown format, `packages` must be a bracketed list.)
```

Comment:
Is it comma-separated? (E.g. nix uses brackets but no commas)

Reply:
I also suggest removing the parenthesis.

Reply:
I also realized that this phrase mak s me thing that the "old format" doesn't support markdown at all, which is false. It seems like more with needs to be done with the explanation. Maybe referencing the changelog-d own doc would be more appropriate. But in any case I find the current version confusing.

Reply:
I wanted to link the `README.md` but the description of "markdown-frontmatter" format isn't a separate link.

Reply:
It's brackets and commas.